tests/console/yast2_clone_system.pm

```diff
@@ -21,7 +21,13 @@ sub run() {
 
     # Install for TW and generate profile
     zypper_call "in autoyast2";
-    assert_script_run "yast2 clone_system", 200;
+    script_run("yast2 clone_system; echo yast2-clone-system-status-\$? > /dev/$serialdev", 0);
+
+    # workaround for bsc#1013605
+    if (check_screen('dhcp-popup')) {
```

Comment:
do not use empty `check_screen` with default timeout. Better use `assert_screen` with alternatives